packages/gatsby/src/redux/actions/public.js

```diff
@@ -944,6 +946,24 @@ actions.createParentChildLink = (
  * @param {Object} config partial webpack config, to be merged into the current one
  */
 actions.setWebpackConfig = (config: Object, plugin?: ?Plugin = null) => {
+  if (config.module?.rules) {
+    config.module.rules.forEach(rule => {
+      if (!rule.resolve) {
+        // TODO move message to gatsbyjs.com/docs - change to structured
+        const key = `${plugin.name}-setWebpackConfig`
```

Comment:
NIT: we access `plugin.name` without checking if `plugin` is defined

Reply:
shouldn't this always be true? If not our CI should fail, right?

Reply:
I'm not sure - the old flow type suggest it's optional